src/test/java/org/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java

```diff
@@ -42,7 +43,7 @@ public void setUp() throws Exception {
         diezSliceTheoremEntry.setField(StandardField.EPRINT, "1812.04698");
         diezSliceTheoremEntry.setField(StandardField.JOURNAL, "arXiv e-prints");
         diezSliceTheoremEntry.setField(StandardField.KEYWORDS, "Mathematics - Differential Geometry, Mathematical Physics, 58B25, (58D19, 58B20, 22E99, 58A35)");
-        diezSliceTheoremEntry.setField(StandardField.MONTH, "Dec");
+        diezSliceTheoremEntry.setMonth(Month.DECEMBER);
```

Comment:
I'm a bit undecided about this one, but we had the standing convention that feature tests should specify the data in the returned entry as explicit as possible (i.e. we really want to test that the data is in the right format in the right field). Since the setMonth performs a bit of magic, this change is against this convention.
On the other hand, it's a bit more readable / invariant under changes.  Nonetheless I would slightly prefer a `setField` solution.

Reply:
I would have argued for semantical equivalence of the returned BibTeX - not for syntactical equivalence.